runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePanel.java

```diff
@@ -54,11 +59,16 @@
 
 	private JTabbedPane tabbedPane = new JTabbedPane();
 
+	private MaterialTabGroup tabGroup;
+	private MaterialTab offersTab;
+	private MaterialTab searchTab;
```

Comment:
These should be final, and hmm, probably also inline variables, not instance ones.

Reply:
THis is still not marked as final